internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -7090,6 +7090,498 @@ func TestHandleGetVoteInfo(t *testing.T) {
+				Hex: "483045022100b29ff29f99ae5b8e3fad72efe6dd13968b7d1c6b3fe2e0fb7eb7656cf7de75f202200888c36da42a0cc948770e3be29b0c75465096bc47cbac27f7c19be7b33287760121039e58379edbbc239e965d7715a9834f6870d9e5e6bf7ebae8d12a30f7282ea5a5",
+			},
+		}},
+		Vout: []types.Vout{{
+			Value:   0.00726867,
+			N:       0,
+			Version: 0,
+			ScriptPubKey: types.ScriptPubKeyResult{
+				Asm:       "OP_DUP OP_HASH160 762432e9619f5ddaf122ac663684152ffe9eb0ec OP_EQUALVERIFY OP_CHECKSIG",
+				Hex:       "76a914762432e9619f5ddaf122ac663684152ffe9eb0ec88ac",
+				ReqSigs:   1,
+				Type:      "pubkeyhash",
+				Addresses: []string{"DsbjabD32RuS1deAj2uTjKfFZ6nSza5qVf3"},
+			}}, {
+			Value:   144.27441143,
+			N:       1,
+			Version: 0,
+			ScriptPubKey: types.ScriptPubKeyResult{
+				Asm:       "OP_DUP OP_HASH160 762432e9619f5ddaf122ac663684152ffe9eb0ec OP_EQUALVERIFY OP_CHECKSIG",
+				Hex:       "76a914762432e9619f5ddaf122ac663684152ffe9eb0ec88ac",
+				ReqSigs:   1,
+				Type:      "pubkeyhash",
+				Addresses: []string{"DsbjabD32RuS1deAj2uTjKfFZ6nSza5qVf3"},
+			}}, {
+			Value:   68.00443066,
+			N:       2,
+			Version: 0,
+			ScriptPubKey: types.ScriptPubKeyResult{
+				Asm:       "OP_DUP OP_HASH160 bc3c059489f447afbf542ff33432adb9ded7f8e9 OP_EQUALVERIFY OP_CHECKSIG",
+				Hex:       "76a914bc3c059489f447afbf542ff33432adb9ded7f8e988ac",
+				ReqSigs:   1,
+				Type:      "pubkeyhash",
+				Addresses: []string{"Dsi8CRt85xYyempXs7ZPL1rBxvDdAGZmgsg"},
+			},
+		}},
+		BlockHash:     "00000000000000001fc4c4c7a3f2ec6d552dda16a3a928f27bd6bd16d8f1e9b3",
+		BlockHeight:   432100,
+		BlockIndex:    11,
+		Confirmations: 1,
+		Time:          1584248018,
+		Blocktime:     1584248018,
+	}
+
+	verboseMempoolResult := verboseResult
+	verboseMempoolResult.BlockHash = ""
+	verboseMempoolResult.BlockHeight = 0
+	verboseMempoolResult.BlockIndex = 0
+	verboseMempoolResult.Confirmations = 0
+	verboseMempoolResult.Time = 0
+	verboseMempoolResult.Blocktime = 0
+
+	tx0TestTx := testTx{
+		hex: hexFromFile("tx432098-11.hex"),
+		indexEntry: &indexers.TxIndexEntry{
+			BlockRegion: database.BlockRegion{
+				Hash: mustParseHash("00000000000000001fc4c4c7a3f2ec6d552dda16a3a928f27bd6" +
+					"bd16d8f1e9b3"),
+				Offset: 52508,
+				Len:    453,
+			},
+			BlockIndex: 11,
+		},
+	}
+
+	var tx wire.MsgTx
+	err := tx.FromBytes(hexToBytes(tx0TestTx.hex))
+	if err != nil {
+		t.Fatalf("unable to create tx from bytes: %v", err)
+	}
+
+	testRPCServerHandler(t, []rpcTest{{
+		name:    "handleGetRawTransaction: invalid txid",
+		handler: handleGetRawTransaction,
+		cmd: &types.GetRawTransactionCmd{
+			Txid:    "invalid",
+			Verbose: &nonVerboseTx,
+		},
+		wantErr: true,
+		errCode: dcrjson.ErrRPCDecodeHexString,
+	}, {
+		name:    "handleGetRawTransaction: tx index not enabled",
+		handler: handleGetRawTransaction,
+		cmd: &types.GetRawTransactionCmd{
+			Txid:    txid,
+			Verbose: &nonVerboseTx,
+		},
+		setTxIndexerNil: true,
+		mockTxMempooler: func() *testTxMempooler {
```

Comment:
There are several of these (and other) cases which repeat the same func implementations.  I would suggest defining said funcs before `testRPCServerHandler` and just using them rather than repeating code a bunch.